components/fl-orchestrator/Makefile

```diff
@@ -33,11 +36,19 @@ install:
 .PHONY: install
 
 clean:
+	rm -rf bin
 	rm -rf dist
 .PHONY: clean
 
-protobuf:
-	protoc --go_out=. --go_opt=paths=source_relative \
-		--go-grpc_out=. --go-grpc_opt=paths=source_relative \
-		pkg/proto/fl_orchestrator.proto
-.PHONY: protobuf
+bin/protoc:
+	./scripts/download-protoc.sh $(PROTOC_VERSION)
+
+bin/protoc-gen-go:
+	GOBIN="$(GOBIN)" go install google.golang.org/protobuf/cmd/protoc-gen-go
+
+bin/protoc-gen-go-grpc:
+	GOBIN="$(GOBIN)" go install google.golang.org/grpc/cmd/protoc-gen-go-grpc
+
+generate: bin/protoc bin/protoc-gen-go bin/protoc-gen-go-grpc
+	PATH="$(GOBIN):$(PATH)" go generate ./pkg/...
```

Comment:
I think it could come in handy moving this `PATH="$(GOBIN):$(PATH)"` at the top of the file, then all other Makefile targets can use the binaries in `bin/`.

Reply:
See above why this wouldn't work.